content/en/docs/overview/singleton-sdk/Angular Client Introduction/translate-pipe.md

```diff
@@ -21,44 +21,66 @@ Get translation by key, will find the source in the sourceBundle according to th
 
 ```
 
-##### **L10n pipe** (deprecated)
-
-```
-{{ key | translate: [ : source [ : variables ] ] }}
+#### **Input**
 
-```
+| Parameter |  Type   | Required | <div style="text-align:center">Description</div>                                                                                                       |
+|:---------:|:-------:|----------|:-------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    key    | string  | required | Define the key to identify the translation, it should name like this: component_module_page_control_shortmsg. e.g. web_settings_stats_statsTable_host; |
+| variables | Array[] | optional | Parameter array, it's used to format the translation/source when they contain placeholders.                                                            |                                        |
 
-#### **Input**
 
-| Parameter |  Type  |                         Description                          |
-| :-------: | :----: | :----------------------------------------------------------: |
-|    key    | string | Define the key to identify the translation, it should name like this: component_module_page_control_shortmsg. e.g. web_settings_stats_statsTable_host; |
+#### **Example Code**
 
-#### **Parameters**
+```
+export const ENGLISH = {
+    'some.unique.key' : 'English for some.unique.key',
+    'VIP.description' : '{0} is common lib developed by VMware G11n team.',
+    'innerHTML.usage' : 'For more details, please refer to <a href=&quot;https://ngx.eng.vmware.com/@vmw/ngx-vip/translate-pipe/documentation&quot;>ngx-vip guide</a>.'
```

Comment:
The link should point to https://github........  you can take a look at it, 

;>Singleton Angular client guide</a>.'

Reply:
Done!